New Resource/Data-Source: HCP Waypoint Application (#794)
hashicorp/terraform-provider-hcp: docs/resources/waypoint_application.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "hcp_waypoint_application Resource - terraform-provider-hcp"
subcategory: ""
description: |-
  Waypoint Application resource
---

# hcp_waypoint_application (Resource)

> Waypoint Application resource

Reviewer: nit: The application data source description ended with a `.`, so I'm suggesting this here to be consistent 😄 

```suggestion
Waypoint Application resource.
```

Author: That's fair. For future reference, this is generated from the schema. I updated the description in e7fc320

<!-- schema generated by tfplugindocs -->
## Schema

### Required

- `application_template_id` (String) ID of the Application Template this Application is based on.
- `name` (String) The name of the Application.

### Optional

- `project_id` (String) The ID of the HCP project where the Waypoint Application is located.

> - `readme_markdown` (String) Instructions for using the Application (markdownformat supported). Note: this is a base64 encoded string, andcan only be set in configuration after initial creation. Theinitial version of the README is generated from the READMETemplate from source Application Template.

Reviewer: ```suggestion
- `readme_markdown` (String) Instructions for using the Application (markdown format supported). Note: this is a base64 encoded string, and can only be set in configuration after initial creation. The initial version of the README is generated from the README template from source Application Template.
```
